build/specifications/MSpec.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "https://raw.githubusercontent.com/nuke-build/nuke/master/source/Nuke.CodeGeneration/schema.json",
+  "references": [
+    "https://github.com/machine/machine.specifications.runner.console"
+  ],
+  "name": "Mspec",
+  "officialUrl": "https://github.com/machine/machine.specifications",
+  "help": "MSpec is called a 'context/specification' test framework because of the 'grammar' that is used in describing and coding the tests or 'specs'.",
+  "packageId": "Machine.Specifications.Runner.Console",
+  "packageExecutable": "mspec-clr4.exe",
+  "tasks": [
+    {
+      "settingsClass": {
+        "properties": [
+          {
+            "name": "Assemblies",
+            "type": "List<string>",
+            "format": "{value}",
+            "separator": " ",
+            "createOverload": true,
+            "help": "Assemblies with tests to be executed executed."
```

Comment:
Duplicate executed

Reply:
Can you explain, please?

Reply:
Assemblies with tests to be executed executed